.scripts/common.ts

```diff
@@ -111,18 +111,15 @@ function isPackageFolderPath(folderPath: string, packagesToIgnore: string[]): bo
   return result;
 }
 
-export const packagesToIgnore: string[] = [
-  "@azure/cosmos",
-  "@azure/storage-blob",
-  "@azure/storage-file-share",
-  "@azure/storage-queue",
-  "@azure/event-hubs",
-  "@azure/event-processor-host",
-  "@azure/keyvault",
-  "@azure/service-bus",
-  "@azure/template",
-  "testhub"
-];
+export const dataplaneListFilename = "list-dataplane.json";
```

Comment:
Why generate a file and then read it here? Why not just the package list like you are doing in the script that generates the data-plane list?

Reply:
I changed the solution from generating a file to call a TS script which has a function return both the list of package names and list of package's folder names. That way I can even skip the paths to the samples of the dataplane packages.